edm4eic.yaml

```diff
@@ -244,6 +244,22 @@ datatypes:
   ## ==========================================================================
   ## Calorimetry
   ## ==========================================================================
+  edm4eic::SimCalorimeterPulse:
+    Description: "Simulated calorimeter pulse prior to digitization."
+    Author: "D. Anderson, S. Joosten"
+    Members:
+      - uint64_t            cellID          // ID of the readout cell for this pulse.
+      - float               energy          // Total energy for this pulse in [GeV].
+      - float               position        // Position of the hit in world coordinates [mm].
+      - float               time            // Start time for the pulse in [ns].
+      - float               interval        // Time interval between amplitude values [ns].
+    VectorMembers:
+      - float               amplitude       // Pulse amplitude in [GeV], sum of amplitude values equals total energy
+    OneToManyRelations:
+      - edm4hep::SimCalorimeterHit hits     // SimCalorimeterHits used to create this pulse
+      - edm4hep::SimCalorimeterPulse pulses // SimCalorimeterPulses used to create this pulse
+      - edm4hep::MCParticle MCParticle      // MCParticle that caused the pulse
```

Comment:
```suggestion
      - edm4hep::MCParticle particle        // MCParticle that caused the pulse
```

Reply:
See also a similar change pushed through in edm4hep for technical reasons outlined there; https://github.com/key4hep/EDM4hep/commit/cb738b0e2a5a8d4c922aa2a0d49974caf4adcd2e#diff-41291f568217a9755476ae21a12825127ae5a00179b19e867cffd64e3ceea36f